scripts/fvt-tools.sh

```diff
@@ -27,7 +27,27 @@ pick_non_master_node() {
 create_hostnet_pods() {
   for index in ${!NODES[*]}
   do
-    kubectl run hostnet-"${NODES[$index]//\./-}" --labels="fvt-helper=hostnet" --overrides="{\"apiVersion\": \"v1\", \"spec\": {\"hostNetwork\":true, \"nodeSelector\": { \"kubernetes.io/hostname\": \"${NODES[$index]}\" }, \"tolerations\": [{ \"operator\": \"Exists\" }]}}" --image busybox -- /bin/tail -f /dev/null
+    cat <<EOF | kubectl create -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    fvt-helper: hostnet
+  name: hostnet-${NODES[$index]//\./-}
+  namespace: default
+spec:
+  containers:
+  - image: busybox
+    name: hostnet-${NODES[$index]//\./-}
+    command: ["/bin/tail"]
```

Comment:
I'm not sure this makes sense :) Or what do you think?

Reply:
Okay, maybe it would look better like this:
https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#run-a-command-in-a-shell

Reply:
IMHO the tail version is also nice and uses less CPU then the infinity while loop.